app/views/content_items/fields_of_operation.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <ul class="govuk-list">
+          <% @content_item.fields_of_operation.each do |name, path| %>
+              <%= link_to name, path, class: "govuk-link" %>
+              <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-2 govuk-!-margin-bottom-2">
```

Comment:
I was wondering if we should define all this [in the stylesheets](https://github.com/alphagov/government-frontend/tree/main/app/assets/stylesheets/views) under the `.fields-of-operation-index` class which is applied to the div:
```
<div id="page" class="fields-of-operation-index">
```
But it's following the pattern of the recently added corporate_information_page.html.erb and history.html.erb so I think it's ok to leave it for consistency.